src/pages-and-resources/discussions/app-list/FeaturesList.jsx

```diff
@@ -1,19 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Remove, Check } from '@edx/paragon/icons';
+import { Check } from '@edx/paragon/icons';
 import { Collapsible } from '@edx/paragon';
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
 import messages from './messages';
 
-const SupportedFeature = (
-  <span className="mr-3">
-    <Check id="check-icon" className="text-success-500" />
-  </span>
-);
-const NonSupportedFeature = (
-  <span className="mr-3">
-    <Remove id="remove-icon" className="text-light-700" />
-  </span>
+const SupportedFeature = (featureName) => (
+  <>
+    <span className="mr-3">
```

Comment:
How will this only effect the mobile view?

Reply:
FeatureList only renders for mobile view. 
FeatureTable renders for desktop view.